database/templates/database/species_detail.html

```diff
@@ -36,13 +36,13 @@ <h2>Isomer Inchis</h2>
     <h2>Thermo Results</h2>
     <ul class="list-group">
         {% for thermo in thermo_list %}
-            <li class="list-group-item"><a href="{% url 'thermo-detail' thermo.pk %}">{{ thermo.source.name }}</a></li>
+            <li class="list-group-item"><a href="{% url 'thermo-detail' thermo.pk %}">{{ thermo }}</a> Hf(298) = {{ thermo.enthalpy298|floatformat:1 }} J/mol</li>
```

Comment:
Maybe we can use some LaTeX here:
```suggestion
            <li class="list-group-item"><a href="{% url 'thermo-detail' thermo.pk %}">{{ thermo }}</a> $\Delta H_f^\circ (298 K) = {{ thermo.enthalpy298|floatformat:1 }} \frac{J}{mol}$</li>
```